torch/utils/cpp_extension.py

```diff
@@ -400,7 +418,7 @@ def unix_cuda_flags(cflags):
             # overriding the option if the user explicitly passed it.
             _ccbin = os.getenv("CC")
             if (
-                _ccbin is not None 
+                _ccbin is not None
```

Comment:
I guess there was an extra whitespace here. 🤷

Reply:
Do we have a Flake8 lint for trailing whitespace?

Reply:
I don't believe we do, but we probably should